plugins/search-backend-module-techdocs/src/collators/DefaultTechDocsCollatorFactory.ts

```diff
@@ -109,8 +109,10 @@ export class DefaultTechDocsCollatorFactory implements DocumentCollatorFactory {
       new CatalogClient({ discoveryApi: options.discovery });
     this.parallelismLimit = options.parallelismLimit ?? 10;
     this.legacyPathCasing = options.legacyPathCasing ?? false;
-    this.entityTransformer =
-      options.entityTransformer ?? defaultTechDocsCollatorEntityTransformer;
+    // @ts-ignore
+    this.entityTransformer = options.entityTransformer ?? (() => {});
```

Comment:
```suggestion
    this.entityTransformer = options.entityTransformer ?? (() => ({}));
```

Reply:
Need some extra parenthesis here to return an empty object (your function was has a Void return type which is why you needed the ts-ignore and the additional `?? {}` below